packages/jest-preset-wpcs/README.md

```diff
@@ -0,0 +1,35 @@
+# @wordpress/jest-preset-wpcs
```

Comment:
Why not `jest-preset-node` to clarify the difference with the default package? `wpcs` looks like a duplicate of the `@wordpress` namespace

Reply:
Thanks @youknowriad I had originally started out using `jest-preset-node` and then switched to `jest-preset-wpcs` due to the Jest configuration I was using, I've refactored that now to be more generic and (I think) this will work.

I've now renamed the package to `jest-preset-node` 👍